XCalendar.Maui/Converters/StringCharLimitConverter.cs

```diff
@@ -12,7 +12,7 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
                     return null;
                 }
 
-                string stringValue = value.ToString();
+                string stringValue = value?.ToString() ?? string.Empty;
```

Comment:
Null is checked for in the previous 3 lines (12-15) so the value would never be null here.

Would it be expected that passing a null value to this coverter would result in a empty string? I think it would make more sense to return null.

Reply:
@ME-MarvinE Yes!
You are right. So my suggestion was misunderstood once my repo ain't updated. Sorry about it.
Once you fixed it between lines 11 and 13 this error shall not happen.